Makefile

```diff
@@ -83,6 +84,9 @@ rekor-server: $(SRCS)
 test:
 	go test ./...
 
+fuzz: $(GO-FUZZ-BUILD)
+	$(GO-FUZZ-BUILD) ./tests/fuzz/...
+
 clean:
 	rm -rf dist
 	rm -rf hack/tools/bin
```

Comment:
should we add something here to delete the `*fuzz.zip` file?

Reply:
The reason I haven't deleted is that you need that to run `go-fuzz bin=zipfile`. WDYT?

Reply:
I would think that if you run `make clean` it should delete any of those artifacts to restore the state of the local repo to as clean as possible. Other than that this PR LGTM

Reply:
I agree, took care of it. Thanks